### PR TITLE
Bug 1296332, added directions on changing the default config file in the Installing Docker section

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1296332

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @tnguyen-rh @bfallonf @tpoitras please peer review :hamster: 

Note that this is labeled for 3.1 only per Peter's comment in the BZ:
"It looks like the ansible installer in 3.2 adds this entry now, so it's no longer a requirement pre-install."

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -940,6 +940,13 @@ and running by default.
 OPTIONS='--selinux-enabled --insecure-registry 172.30.0.0/16'
 ----
 +
+To easily script a complete installation from a kickstart or cloud-init setup,
+change the default configuration file:
++
+----
+# sed -i '/OPTIONS=.*/c\OPTIONS="--selinux-enabled --insecure-registry 172.30.0.0/16"' /etc/sysconfig/docker
```
Comment (tnguyen-rh) [inline install_config/install/prerequisites.adoc]: Suggest breaking this line (w/ backslash-newline) right before the filename.

**Comment by tnguyen-rh** [comment]
@ahardin-rh A small suggestion for the sake of the (ridiculously) narrow customer portal rendering, otherwise LGTM.

**Comment by bfallonf** [comment]
@ahardin-rh LGTM :+1:

**Comment by tpoitras** [comment]
@ahardin-rh all good, looks good, good stuff. :minibus:

**Pull request merged**